JavaScript/JavaScriptSDK.Tests/E2ETests/snippet.tests.ts

```diff
@@ -106,6 +106,35 @@ class SnippetTests extends TestClass {
                 }
             ]
         });
+
+        var sendSpy;
+        this.testCaseAsync({
+            name: "SnippetTests: SDK and Snippet versions are handled correctly",
+            stepDelay: 250,
+            steps: [
+                () => {
+                    this.loadSnippet(path70);
+                },
+                () => {
+                    sendSpy = this.setAppInsights();
+
+                    window["appInsights"].trackPageView("test", { property: "p1" }, { measurement: 5 });
+                },
+                () => {
+                    Assert.equal(this.queueCallCount, this.queueSpy.callCount, "queue is emptied");
+                    Assert.equal(1, sendSpy.sender.callCount, "v2 send called");
```

Comment:
Aren't the messages in asserts shown when the assert fails?  They read like they are success messages so maybe I misunderstand the expected behavior.